types/webpack/test/index.ts

```diff
@@ -1166,3 +1166,14 @@ class LoggingPlugin extends webpack.Plugin {
         });
     }
 }
+
+// Stats Microsoft/DefinitelyTyped#43952
+import { Stats } from 'webpack';
```

Comment:
IMHO not a big fan of imports in the middle of the file, because we already have a general import of the whole module at the top:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/abe7571daf1b1b864cd6d1790670f1c910d1ea34/types/webpack/test/index.ts#L1

I think it would be better to destruct the `webpack` import here which has the same effect:

```suggestion
const { Stats } = webpack;
```